app/subscriber/src/features/content/view-content/ViewContent.tsx

```diff
@@ -149,7 +149,7 @@ export const ViewContent: React.FC = () => {
         </p>
         <p className="source-section">
           <Show visible={content?.contentType === ContentTypeName.PrintContent}>
-            {content?.source?.name} - {content?.section}
+            {content?.source?.name} - {`${content?.section}: ${content?.page}`}
```

Comment:
Not all will have a section.  Which would result in an extra `:` in the output.

Reply:
Will adjust!